Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -495,13 +481,13 @@ private Subscription CreateUniverseSubscription(SubscriptionRequest request)
                             args.Action == NotifyCollectionChangedAction.Add ? args.NewItems :
                             args.Action == NotifyCollectionChangedAction.Remove ? args.OldItems : null;
 
-                        if (items == null || _frontierUtc == DateTime.MinValue) return;
+                        if (items == null || _frontierTimeProvider.GetUtcNow() == DateTime.MinValue) return;
```

Comment:
let's extract `_frontierTimeProvider.GetUtcNow()` into a local variable so we're not invoking it three times here

Reply:
Whops! Shame on me, I'll update it 👍